### DISCUSSION TITLE
getWalletTransaction

### DISCUSSION THREAD

**Discussion by g00m00nry0ng** [lintangtimur/ovoid#59] [Q&A]
ada yg sudah berhasil menggunakan getWalletTransaction? Barusan saya coba masih dapat response error code: 1020

**Comment by herihermwn** [comment]
History transaksi wallet ya ?

Seperti ini bukan ?
http://188.166.233.97:8094/ovo/history

**Reply by g00m00nry0ng** [reply]
> History transaksi wallet ya ?
> 
> Seperti ini bukan ?
> http://188.166.233.97:8094/ovo/history

Bener Gan. Kalau berkenan boleh nyontek php script nya Gan. 😁🙏

**Reply by herihermwn** [reply]
Kurang lebih begini cara requestnya,

### Http method
- Get

### Url 
- https://api.ovo.id/wallet/v2/transaction

### Header 
    {
        "client-id": "ovo_android",
        "device-id": device_id,
        "authorization": refresh_token,
        "Connection": "close",
        "Accept-Encoding": "gzip, deflate",
        "User-Agent": "okhttp/4.9.0",
        "Content-Type": "application/json"
    }

**Reply by raf-lii** [reply]
saya kok dapat respon seperti ini ya?
`{"status":500,"data":[],"message":"Maaf transaksi anda sedang tidak dapat diproses"}`

**Reply by herihermwn** [reply]
> saya kok dapat respon seperti ini ya?
> `{"status":500,"data":[],"message":"Maaf transaksi anda sedang tidak dapat diproses"}`

![image](https://user-images.githubusercontent.com/57160005/133878723-b83654d9-319f-4127-a86d-1cf594bd77c9.png)
Kalau dibuka tanpa token outputnya sama, coba pastiin lagi tokennya valid

**Reply by raf-lii** [reply]
> > saya kok dapat respon seperti ini ya?
> > `{"status":500,"data":[],"message":"Maaf transaksi anda sedang tidak dapat diproses"}`
> 
> ![image](https://user-images.githubusercontent.com/57160005/133878723-b83654d9-319f-4127-a86d-1cf594bd77c9.png)
> Kalau dibuka tanpa token outputnya sama, coba pastiin lagi tokennya valid

tokennya sudah valid tpi kenapa mendapatkan output yang sama seperti diatas ya?

**Reply by herihermwn** [reply]
> > > saya kok dapat respon seperti ini ya?
> > > `{"status":500,"data":[],"message":"Maaf transaksi anda sedang tidak dapat diproses"}`
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/57160005/133878723-b83654d9-319f-4127-a86d-1cf594bd77c9.png)
> > Kalau dibuka tanpa token outputnya sama, coba pastiin lagi tokennya valid
> 
> tokennya sudah valid tpi kenapa mendapatkan output yang sama seperti diatas ya?

Waduh kurang tau deh, tokennya sudah dicoba di API lain ?
